mvpa2/tests/test_searchlight.py

```diff
@@ -762,6 +762,20 @@ def measure(ds):
             for f in glob.glob(tfile + '*'):
                 os.unlink(f)
 
+    def test_adhoc_searchlight_pass_attr(test):
+        ds1 = datasets['3dsmall'].copy(deep=True)
+        # GNB searchlight with raw predictions
+        gnb_sl = GNBSearchlight(
+            GNB(),
+            generator=NFoldPartitioner(count=2),
+            qe=IndexQueryEngine(myspace=Sphere(2)),
+            errorfx=None,
+            pass_attr=ds1.sa.keys())
+        res = gnb_sl(ds1)
+        for k in ds1.sa.keys():
+            assert_in(k, res.sa.keys())
```

Comment:
heh, this test has no check whatsoever if concatenation worked out correctly.... you could have assigned some "sa.sample_id" with xrange and verify that obtained .sample_id is as it should be given ds.sa.chunks and the nfold partitioner with count=2